styleguide/backstop.json

```diff
@@ -55,6 +55,19 @@
       ],
       "misMatchThreshold" : 1,
       "requireSameDimensions": false
+    },
+    {
+      "label": "press-release-page",
+      "url": "http://localhost:3000/?p=pages-press-release",
```

Comment:
Please update this to the "in a new window" URL, some of the screenshots are getting cut off because it can't scroll for the screenshot. This will also require that you re-run `backstop approve` to create the updated screenshots.